src/checkqueue.h

```diff
@@ -96,7 +98,11 @@ class CCheckQueue
                         return fRet;
                     }
                     nIdle++;
+                    if (interrupted)
+                        return false;
                     cond.wait(lock); // wait
+                    if (interrupted)
+                        return false;
```

Comment:
> If an `if` only has a single-statement `then`-clause, it can appear
    on the same line as the `if`, without braces. In every other case,
    braces are required, and the `then` and `else` clauses must appear
    correctly indented on a new line.
https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c